doc/icub-whole-body-torque-control-faqs.md

```diff
@@ -6,3 +6,102 @@ Frequent Asked Questions on whole-body torque control on the iCub robot  {#icub-
 ### What are the things to check on the iCub robot before performing whole-body torque control experiments? 
 
 
+# Do a complete update of information (before the calibration)
+
+## Usual update before processing
+First, update the new version of yarp/icub_main/codyco-
+Then, if there are some updates, you have to build them (make) on all the computers (because builds are not shared) with the usual:  
+> cd build  
```

Comment:
If you enclose the bash code blocks with three tildes before and after, you get a nice rendering of the code, as in:
~~~
cmake .
make 
~~~
To discover more on this, see https://guides.github.com/features/mastering-markdown/ .